specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json

```diff
@@ -0,0 +1,4472 @@
+            "$ref": "./examples/DeviceManagement_ListLogCollections.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/InstanceIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of all log collections",
+            "schema": {
+              "$ref": "#/definitions/LogCollectionList"
+            }
+          },
+          "default": {
+            "description": "Default response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/deviceUpdate/{instanceId}/management/deviceDiagnostics/logCollections/{operationId}/detailedStatus": {
+      "get": {
+        "tags": [
+          "Device and deployment management"
+        ],
+        "operationId": "DeviceManagement_GetLogCollectionDetailedStatus",
+        "description": "Get log collection with detailed status",
+        "x-ms-examples": {
+          "DeviceManagement_GetLogCollectionDetailedStatus": {
+            "$ref": "./examples/DeviceManagement_GetLogCollectionDetailedStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/InstanceIdParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Device diagnostics operation with detailed status",
+            "schema": {
+              "$ref": "#/definitions/LogCollectionOperationDetailedStatus"
+            }
+          },
+          "default": {
+            "description": "Default response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/deviceUpdate/{instanceId}/management/deviceDiagnostics/deviceHealth": {
+      "get": {
+        "tags": [
+          "Device and deployment management"
+        ],
+        "operationId": "DeviceManagement_ListDeviceHealth",
+        "description": "Get list of device health",
+        "x-ms-examples": {
+          "DeviceManagement_ListDeviceHealth": {
+            "$ref": "./examples/DeviceManagement_ListDeviceHealth.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/InstanceIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "filter",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Filter list by specified properties."
```

Comment:
Indicating the supported filter syntax would be helpful. Some services use OData-like `$filter`, here we dropped the leading '$' but I presume it's using a similar syntax (e.g. `prop eq bar`)?

Are any of the other conventional filter functions (e.g. [those supported by Graph](https://docs.microsoft.com/en-us/graph/filter-query-parameter)) supported or just `eq` equality?